src/Concordium/Client/Types/GRPC.hs

```diff
@@ -0,0 +1,20 @@
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Concordium.Client.Types.GRPC (
+    GRPCResult
+  , GRPCResponse(..)
+  , GRPCHeaderList
+  ) where
+
+import Network.GRPC.Client
+
+data GRPCResponse a = GRPCResponse
+  { grpcHeaders :: CIHeaderList,
+    grpcResponse :: a
+  } deriving Show
+
+instance Functor GRPCResponse where
+  fmap f (GRPCResponse hds v) = GRPCResponse hds $ f v
+
+type GRPCResult a = Either String (GRPCResponse a)
+type GRPCHeaderList = CIHeaderList
```

Comment:
Add this go `GRPC` directly and remove this module. We already have many types there so let's keep that.

Reply:
I did this initially but this introduces a cyclic dependency between GRPC and Helper. Is there some way to avoid that?

Reply:
I see.

We could add it to Helper then, or move the helper definitions that need this to GRPC. We only need those internally anyhow.